lib/maze/option.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Provides the set of Maze Runner command line options
+module Maze
+  class Option
```

Comment:
Would making this a `Maze::Option` module, then renaming the classes to `Maze::Option::Parser`, `Maze::Option::Processor` & `Maze::Option::Validator` make sense?

Reply:
Yes, completely - done.